SPEC.md

```diff
@@ -543,6 +543,8 @@ Accessible at `$AC_METADATA_URL/acMetadata/v1/pod/hmac`
 
 ### AC Name Type
 
+Field `acName`
```

Comment:
what are you saying? just remove acName here? since it's used else where as a formatting strictness for field values